### ISSUE TITLE
Calendar AM/PM stepper only triggers model change one time

### ISSUE THREAD

**Issue by ohsomofo** [primefaces/primeng#3056]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
Please fork the plunkr below and create a case demonstrating your bug report. Issues without a plunkr have much less possibility to be reviewed.

http://plnkr.co/edit/VZbdZfQ7SuTV87tTkbYR?p=preview

**Current behavior**
When toggling AM/PM with stepper the underlying date object is only changed the first time you use the am/pm stepper. After that you can repeatedly click the up or down arrows and the model will remain unchanged. 

**Expected behavior**
When using the AM/PM stepper the underlying model object should be updated to reflect the currently selected time each time you use the AM/PM stepper.

**Minimal reproduction of the problem with instructions**

Example showing issue using primeng 4.0.3
http://plnkr.co/edit/mEctar?p=preview

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

MacOS
angular/cli: 1.0.1
node: 7.7.4
os: darwin x64
angular/animations: 4.1.0
angular/common: 4.1.0
angular/compiler: 4.1.0
angular/core: 4.1.0
angular/forms: 4.1.0
angular/http: 4.0.3
angular/platform-browser: 4.1.0
angular/platform-browser-dynamic: 4.1.0
angular/router: 4.1.0
angular/cli: 1.0.1
angular/compiler-cli: 4.1.0

* **Angular version:** 4.1.0

* **PrimeNG version:** 4.0.3

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

 Safari 10.1.1
Chrome Version 58.0.3029.110 (64-bit)

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =

**Comment by mlakhara** [comment]
Looks like this was fixed in version 4.0.1 but again seemed to make its way back. I just updated from 4.0.1 to 4.1.0-rc.3 to find this.

**Comment by yigitfindikli** [comment]
Unable to replicate with latest PrimeNG, if the issue persists please create a new ticket with a test case reproducing the issue e.g. stackblitz or a github repo and it will be reviewed by our team once again.
